### DISCUSSION TITLE
Keine Daten über MQTT

### DISCUSSION THREAD

**Discussion by romanstopka72** [softwarecrash/EPEver2MQTT#26] [General]
Hallo !
Ich habe alles wie beschrieben aufgesetzt aber über MQTT bekomme ich keine Daten .
Es kommt nur über den Topic Data {"DEVICE_NAME":"EPEver2MQTT"} rein.

**Comment by softwarecrash** [comment]
Hallo,
ein paar informationen brauchen wir schon noch
- verkabelung
- bilder
- eingesetzte hardware
- max chip bezeichnung

**Reply by romanstopka72** [reply]
-EPEVER MPPT Laderegler 30A Tracer3210AN

Hier die Fotos:
![20230910_054341](https://github.com/softwarecrash/EPEver2MQTT/assets/64645083/b7d18d8f-4d04-4c1b-becf-b19d0b6a67b2)
![20230910_054142](https://github.com/softwarecrash/EPEver2MQTT/assets/64645083/551f97a6-6db0-48b8-bac0-ceeaef7f75df)
![20230910_054304](https://github.com/softwarecrash/EPEver2MQTT/assets/64645083/1caf7dd5-a6f2-4e58-9e04-d7dd3e9408bc)
![20230910_054412](https://github.com/softwarecrash/EPEver2MQTT/assets/64645083/dcd7a2bf-3d36-4a9c-818e-6f47ca25aec0)
![20230910_054439](https://github.com/softwarecrash/EPEver2MQTT/assets/64645083/23a8593a-b567-4197-a78e-8c2406b4ba41)

**Reply by softwarecrash** [reply]
DE-RE hat verbindung?
kannst nur mal Versuchen A+ und B- zu vertauschen.
ansonsten lässt sich da erstmal nicht viel erkennen.
der  EP sollte funktionieren, kann eigentlich nur an der verkabelung oder der RS platine liegen.

**Comment by romanstopka72** [comment]
DE-RE: ja

Wie schauts mit dem aus:

ARCELI 5 Stück TTL zu RS485 485 zu seriellem UART Level Reziproke Hardware
Automatische Flusskontrolle UART zu RS485 Konverter RS485 zu TTL
https://amzn.eu/d/g3jQK2f


Danke





Rocky ***@***.***> schrieb am So., 10. Sep. 2023, 11:22:

**Reply by softwarecrash** [reply]
die sind meist murks, da ein zweiter serial chip drauf, die machen hin und wieder probleme, das was du da hast sollte im regelfall funktionieren.
hast du mal versucht die ID neu zu setzen? `http://<ESP-IP>/set?devid=1`
wenn du zwar nur einen hast der aber nicht auf ID1 ist dann kommt auch nichts

**Comment by romanstopka72** [comment]
Hab das probiert. Es kommt:

ID set Fail... Actual id is: 0

oder bei 0:

ID 0 Successfull Set


Am So., 10. Sept. 2023 um 13:09 Uhr schrieb Rocky ***@***.***

**Reply by softwarecrash** [reply]
dann besteht keine kommunikation mit dem EP

**Comment by romanstopka72** [comment]
Ich habe gestern noch ein bisschen probiert, aber nix gefunden.

Was ich auch probiert habe, ist der andere UART zu RS485 Konverter.
Da hat er die ID wechsel übernommen. Hat aber auch nix gebracht.
Im Log hat er immer "get Time error" und "get live data error" angezeigt.

Hab jetzt noch das Epever USB Kabel bestellt und schau ob ich was in den
Computer bekomme.
Ein zweiter Laderegler ist auch am Weg.
Vielleicht liegt's am Laderegler.

Gibt es eigentlich eine Möglichkeit den Datenausgang zu kontrollieren ?


Am Mo., 11. Sept. 2023 um 19:50 Uhr schrieb Rocky ***@***.***

**Reply by softwarecrash** [reply]
es gibt modbus zu usb adapter damit kann man es prüfen und auch den EP einstellen.
ggf ist der rj45 zu stiftleiste adapter den du da hast irgendwie "falsch" rum gepolt.

**Comment by romanstopka72** [comment]
Ich werde noch ein Netzwerkkabel auftrennen und probieren.


Am Di., 12. Sept. 2023 um 12:00 Uhr schrieb Rocky ***@***.***

**Comment by romanstopka72** [comment]
Hallo !

Jetzt hat es funktioniert.
Verkabelung wie von mir gemacht und ein neuer Laderegler.
Laderegler geht zurück.

Danke nochmal für deine Hilfe.

Grüße
Roman

Am Di., 12. Sept. 2023 um 12:48 Uhr schrieb Roman Stopka <
***@***.***>:
